cdap-ui/app/cdap/components/DataPrepConnections/index.js

```diff
@@ -92,14 +92,14 @@ export default class DataPrepConnections extends Component {
               <span>
                 {T.translate(`${PREFIX}.upload`)}
               </span>
-            </Link>
+            </NavLink>
           </div>
 
           <div className="menu-item">
-            <Link
+            <NavLink
               to={`${baseLinkPath}/browser`}
               activeClassName="active"
-              isActive={(location) => {
+              isActive={(match, location) => {
```

Comment:
Since we now have `match` I think the regex can simplified (or removed?) as match will be a falsy value if react-router was not able to match. `location.pathname` is the same as `match.path` I believe. Reference: https://reacttraining.com/react-router/web/api/NavLink/isActive-func